core/CliMulti/Process.php

```diff
@@ -297,6 +297,9 @@ public static function getRunningProcesses()
      {
          $ids = explode("\n", trim(`ps x 2>/dev/null | awk '! /defunct/ {print $1}' 2>/dev/null`));
 
+         print getmypid() . "\n";
+         print `ps x 2>/dev/null | awk '! /defunct/ {print $1}' 2>/dev/null` . "\n";
+
```

Comment:
Did you add that on purpose?

Reply:
No, task switching w/ an L3 issue, will remove